### ISSUE TITLE
if i want train and test in my own dataset，should i pretain a own boundarymodel？

### ISSUE THREAD

**Issue by shaonianG** [fz-20/BGPSeg#5]
if i need，how to train a myself pretrain-model？
look for your reply，thank you

**Comment by fz-20** [comment]
Sorry for the late reply.
Yes, if you want to train and test on your own dataset and you have boundary point annotations, it is recommended to pretrain your own boundary model. For pretraining, you can directly use the [BoundaryNet](https://github.com/fz-20/BGPSeg/blob/cbb7431efb739b2ad030ff8e8047c7d098398a4f/model/BGPSeg.py#L210) and pretrain it with a simple weighted binary cross-entropy loss. Hope this helps.
